src/shared/utils/billing.test.ts

```diff
@@ -535,32 +488,3 @@ describe('isTrialPlan', () => {
     expect(isTrialPlan(Plans.USERS_BASIC)).toBeFalsy()
   })
 })
-
-describe('isProPlan', () => {
```

Comment:
Are we adding the logic in these tests somewhere?

Reply:
The logic now resides in API!